libbeat/docs/processors-using.asciidoc

```diff
@@ -1184,6 +1184,28 @@ The following settings are supported:
                   empty array (`[]`) or an empty object (`{}`) are considered
                   empty values. Default is `false`.
 
+[[fingerprint]]
+=== Generate a fingerprint of an event
+
+The `fingerprint` processor generates a fingerprint of an event based on a
+specified subset of its fields.
+
+[source,yaml]
+-----------------------------------------------------
+processors:
+ - fingerprint:
+     fields: ["field1", "field2", ...]
+-----------------------------------------------------
+
+.Fingerprint options
+[options="header"]
+|======
```

Comment:
You forgot the header row here. :-)  The rendered table looks like this: 

![image](https://user-images.githubusercontent.com/14206422/67908010-107e5a80-fb37-11e9-8431-148dd05ef2e4.png)